modules/release.nix

```diff
@@ -211,7 +212,7 @@ in
       echo Building factory image
       ${factoryImgScript { targetFiles=signedTargetFiles.name; img=img.name; out=factoryImg.name; }}
       echo Writing updater metadata
-      ${writeOtaMetadata "."}
+      ${writeOtaMetadata { targetFiles=unsignedTargetFiles; path = "."; }}
```

Comment:
Given that we just created the OTA itself in the releaseScript on line 201, couldn't we just pass the path to it (`ota.name`) as an argument to `writeOtaMetadata`--instead of the target files?. Then it'd be exact and not a calculation.

Reply:
Makes sense. Done.